support-services/src/main/scala/com/gu/support/promotions/PromotionService.scala

```diff
@@ -1,25 +1,49 @@
 package com.gu.support.promotions
 
-import com.gu.i18n.Country
-import com.gu.support.catalog.ProductRatePlanId
+import com.gu.i18n.{Country, CountryGroup}
+import com.gu.support.catalog.{GuardianWeekly, ProductRatePlanId}
 import com.gu.support.config.PromotionsConfig
 import com.gu.support.promotions.PromotionValidator.PromotionExtensions
 import com.gu.support.touchpoint.TouchpointService
 import com.gu.support.zuora.api.SubscriptionData
 import com.typesafe.scalalogging.LazyLogging
+import org.joda.time.DateTime
 
 class PromotionService(config: PromotionsConfig, maybeCollection: Option[PromotionCollection] = None) extends TouchpointService with LazyLogging {
   val promotionCollection = maybeCollection.getOrElse(new DynamoPromotionCollection(config.tables))
 
+  val SixForSixPromotion = Promotion(
+    "Six For Six",
```

Comment:
I'd find named parameters a little easier to follow, but understand if this is temporary.